boards/Makefile

```diff
@@ -110,19 +111,24 @@ endif
 
 depend: .depend
 
-$(DUMMY_KCONFIG): $(BOARD_KCONFIG)
-	$(call DELFILE, $(DUMMY_KCONFIG))
-	$(call COPYFILE, $(BOARD_KCONFIG), $(DUMMY_KCONFIG))
+$(DUMMY_BOARD_KCONFIG): $(BOARD_KCONFIG)
+	$(call DELFILE, $(DUMMY_BOARD_KCONFIG))
+	$(call COPYFILE, $(BOARD_KCONFIG), $(DUMMY_BOARD_KCONFIG))
 
-dirlinks: $(DUMMY_KCONFIG)
+$(DUMMY_CHIP_KCONFIG): $(CHIP_KCONFIG)
+	$(call DELFILE, $(DUMMY_CHIP_KCONFIG))
+	$(call COPYFILE, $(CHIP_KCONFIG), $(DUMMY_CHIP_KCONFIG))
 
-context: $(DUMMY_KCONFIG)
+dirlinks: $(DUMMY_BOARD_KCONFIG) $(DUMMY_CHIP_KCONFIG)
+
+context: $(DUMMY_BOARD_KCONFIG) $(DUMMY_CHIP_KCONFIG)
```

Comment:
Is this the right place to add an arch context operation?  These two (board and arch context) are similar but two different operations.
I added a way to include arch into `context` but it's stranded inside #1893

Reply:
@Ouss4 your are right, but I consider update it into Makefile.unix/Makefile.win to avoid duplicate dummy code in all archs context. How about it?

Reply:
Still some build break, I'll update it later.

Reply:
> @Ouss4 your are right, but I consider update it into Makefile.unix/Makefile.win to avoid duplicate dummy code in all archs context. How about it?

I think we can have it there, that looks better.

> Still some build break, I'll update it later.

Is the build break related to this PR?

Reply:
Yes,  it's related since arch/dummy/Kconfig not found. I have forced push again.